repo/packages/E/elasticsearch/0/marathon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "elasticsearch",
+  "cpus": 0.2,
```

Comment:
It is required that this is a property configured from config.json.  This allows customization of the value if desired.